### PR TITLE
FISH-105 Document change in migrate timers

### PR DESCRIPTION
**Pull request by Pandrex247**
Timers can now be migrated from live instances.

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/payara-server/ejb/persistent-timers.adoc [hunk 1]
```diff
@@ -62,3 +62,5 @@ asadmin> migrate-timers --target server1 server2
 ----
 
 Where `server1` is the active instance to migrate timers to and `server2` is the failed instance.
+
```
Comment (fturizo) [inline docs/modules/ROOT/pages/documentation/payara-server/ejb/persistent-timers.adoc:65]: Looks good, but due to how relevant and important the feature is, I'd put this statement in a separate section like this:

[[migration-live-instance]]
=== Migration from Live Instances

_Since Payara Server 5.2020.1_

The `migrate-timers` command can also be used to migrate timers that are scheduled to expire on a live instance. (WOULD BE GOOD TO ADD SOME INFO ON WHAT THIS MEANS AS WELL)

**Review by fturizo — CHANGES_REQUESTED** [review]
Minor change needed

File: docs/modules/ROOT/pages/documentation/payara-server/ejb/persistent-timers.adoc (hunk 1 above)
Comment (Pandrex247) [reply docs/modules/ROOT/pages/documentation/payara-server/ejb/persistent-timers.adoc:65]: Donezo. How's it looking now?

**Review by fturizo — APPROVED** [review]
Looks good

**Pull request merged**